tests/integration-tests/tests/pcluster_api/test_api.py

```diff
@@ -121,26 +125,134 @@ def _ec2_wait(region, instances, waiter_type):
     waiter.wait(InstanceIds=instances)
 
 
+@pytest.fixture(scope="session", name="resources_dir")
+def resources_dir_fixture():
+    return Path(pkg_resources.resource_filename(__name__, "/../../resources"))
+
+
+@pytest.fixture(scope="session", name="policies_template_path")
+def policies_template_path_fixture(resources_dir):
+    return resources_dir / ".." / ".." / ".." / "cloudformation" / "policies" / "parallelcluster-policies.yaml"
+
+
+@pytest.fixture(scope="class", name="custom_actions_bucket_name")
+def custom_actions_bucket_name_fixture(s3_bucket_factory):
+    return s3_bucket_factory()
+
+
+@pytest.fixture(scope="class", name="policies_template_with_custom_actions_bucket_access")
+def policies_template_with_custom_actions_bucket_access_fixture(policies_template_path, custom_actions_bucket_name):
+    with open(policies_template_path, "r", encoding="utf-8") as f:
+        policies_template = TemplateGenerator(cfn_tools.load_yaml(f.read()))
+
+    policy_document = policies_template.resources.get("ParallelClusterClusterPolicy").properties["PolicyDocument"]
+    statement = policy_document.get("Statement")
+    statement.append(
+        {
+            "Action": ["s3:GetObject"],
+            "Effect": "Allow",
+            "Resource": {"Fn::Sub": f"arn:${{AWS::Partition}}:s3:::{custom_actions_bucket_name}/*"},
+        }
+    )
+    statement.append(
+        {
+            "Action": ["events:PutRule", "events:DeleteRule", "events:PutTargets", "events:RemoveTargets"],
```

Comment:
nitpick, non blocking, you should not need this?

Reply:
removed